packages/scandipwa/src/route/ProductPage/ProductPage.component.js

```diff
@@ -129,31 +106,105 @@ export class ProductPage extends PureComponent {
         );
     }
 
-    renderAdditionalSections() {
+    getTabNames() {
+        const tabNames = [];
+        const {
+            isProductInformationTabEmpty,
+            isProductAttributesTabEmpty
+        } = this.props;
+
+        if (!isProductInformationTabEmpty()) {
+            tabNames.push(__('About'));
+        }
+
+        if (!isProductAttributesTabEmpty()) {
+            tabNames.push(__('Details'));
+        }
+
+        tabNames.push(__('Reviews'));
+
+        return tabNames;
```

Comment:
https://github.com/scandipwa/scandipwa/pull/1978#discussion_r549376607 this is not resolved. See [this video](https://scandipwa.gitbook.io/create-scandipwa-app/hands-on-tutorials/patterns-of-scandipwa) to learn more.

Reply:
@alfredsgenkins please update your link, since it is broken. Thanks!

Reply:
https://docs.scandipwa.com/hands-on-tutorials/patterns-of-scandipwa